INSTALL.md

```diff
@@ -49,9 +49,9 @@ Download the latest version of Python 3 from Python.org: https://www.python.org/
 
 #### macOS
 
-By default, macOS comes with Python 2.7. For best usage, need Python 3.6. The easiest way to do so is to install it through `Homebrew`. If you are not familiar with `homebrew`, read more about it here: https://brew.sh/
+By default, macOS comes with Python 2.7. For best usage, need at least Python 3.2.  The easiest way to do so is to install it through `Homebrew`. If you are not familiar with `homebrew`, read more about it here: https://brew.sh/
 
-To install Python 3.6:
+To install Python 3.2:
```

Comment:
I'm ok with the 3.2 above, but I find this line confusing, because one would think it is recommended to install 3.2 and may be confused if brew installs 3.6, don't you tink?

Reply:
How about:

> To install Python >= 3.2:


That would fit for all compatible versions of Python.

Reply:
I decided for slightly different wording, but to a same effect.